src/pudl/transform/eia861.py

```diff
@@ -564,6 +564,15 @@ def _tidy_customer_class_dfs(df, df_name, idx_cols):
     return tidy_df
 
 
+def _zero_pad_zips(zip_series):
+    # Add preceeding zeros where necessary and get rid of decimal zeros
+    zip_series = (
+        pd.Series([str(zipcode).split('.')[0].zfill(5)
+                   for zipcode in zip_series if pd.notnull(zipcode)])
```

Comment:
Maybe you already explored these possibilities in the notebook interactively and they're not an issue, but some things that come to mind looking at this:
* Does this also work when the zipcode hasn't been interpreted as a `float` and there's no decimal point? (In cases where there are no missing values in the column, it'll likely come through as an integer.)
* Are there any non-numeric values lurking in the column initially?
* Do you end up with any NA values that have been converted into strings?
* Are there any invalid all-zero "00000" zip codes? (which are common in the EIA 860 at least).
* We'll need to apply this to both the 5-digit Zip code, and also the 4-digit Zip code suffixes (which also need to be zero-padded).

I think those issues could be addressed with:
```py
# N will need to be variable so we can zero pad Zip, and also the +4 suffixes
def zero_pad_zips(zip_series, N):
    zip_series = (
        pd.to_numeric(zip_series)  # Make sure it's all numerical values
        .astype(pd.Int64Dtype())  # Make it a (nullable) Integer
        .fillna(0)  # fill the NA
        .astype(str).str.zfill(N)  # Make it a string and zero pad it.
        .astype(pd.StringDtype())  # Make it nullable
        .replace({N*"0": pd.NA})  # All-zero Zip codes aren't valid.
    )
    return zip_series
```

Reply:
All important concerns! I'll go ahead and update it.